docs/coins.md

```diff
@@ -41,6 +41,7 @@ This list is generated from [./coins.json](../coins.json)
 | 425     | Aion             | AION   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/aion/info/logo.png" width="32" />         | <https://aion.network>        |
 | 457     | Aeternity        | AE     | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/aeternity/info/logo.png" width="32" />    | <https://aeternity.com>       |
 | 500     | Theta            | THETA  | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/theta/info/logo.png" width="32" />        | <https://www.thetatoken.org>  |
+| 505     | Telegram         | TON    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ton/info/logo.png" width="32" />          | <https://github.com/copperbits/TON> |
```

Comment:
@catenocrypt Make sure to add coin to SLIP-44 registry https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
@kolya182 , you mean I should create a PR there?  What value should I chose? I've just picked a random value...

Reply:
SLIP-44 request created,  https://github.com/satoshilabs/slips/pull/752

Reply:
> @kolya182 , you mean I should create a PR there? What value should I chose? I've just picked a random value...

You picked correct one. Once it merged please make PR to https://github.com/trustwallet/assets with TON chain logo.

Reply:
Slips PR has been merged, TON is included in Slip-44, with value 396.  This PR has been updated to use that value.